apple/DemoApp/Demo/DemoNavigationView.swift

```diff
@@ -148,6 +150,23 @@ struct DemoNavigationView: View {
             .task {
                 await getRoutes()
             }
+            .overlay(
+
+                HStack {
+
+                    Spacer()
+
+                    if isNavigationActive {
+
+                        MuteButton(isMuted: $spokenInstructionObserver.isMuted)
+
+                    }
+
+                },
+
+                alignment: .topTrailing
+
+            )
```

Comment:
This works for the demo app, but doesn't do much for everyone building apps with the library components :)

Have a look at `PortraitNavigationOverlayView` and `LandscapeNavigationOverlayView` for how we handle this. We already built the whole grid layout and have ways to configure what's shown where, as well as overrides. Moving the mute button to the top trailing position there will also conveniently mean it's shown and hidden automatically in the future based on whether the user is navigating.